src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -393,6 +393,13 @@ export abstract class AbstractSettingRenderer extends Disposable implements ITre
 			toggleMenuTitle
 		});
 		toolbar.setActions([], this.settingActions)();
+
+		// change icon from ellipsis to gear
+		let icon = container.querySelector('.codicon-more');
+		if (icon) {
+			(<HTMLElement>icon).classList.add('codicon-gear');
```

Comment:
When does the icon change from ellipsis to gear?

Reply:
This is in the Settings GUI that brings up the context menu:

<img width="233" alt="image" src="https://user-images.githubusercontent.com/35271042/65803627-f9151180-e133-11e9-8a3a-71d05ed47f73.png">

The default is an ellipsis icon from the toolbar widget:
https://github.com/microsoft/vscode/blob/6f070c5a354d85afdcd3657b348d8c4d6a1269d4/src/vs/base/browser/ui/toolbar/toolbar.ts#L61-L71

Reply:
Oh I get it